.changes/wix-per-machine.md

```diff
@@ -0,0 +1,9 @@
+---
+"tauri-bundler": "patch"
+"cli.rs": "patch"
+"cli.js": "patch"
+---
+
+Added `perMachine` option to `tauri.conf.json > tauri > bundle > windows > wix`, disabled by default.
+**Breaking change** `msi` installer will install your app per-user by default now.
```

Comment:
It is a slight breaking change, that's why I kept the bump as `patch` (still considering to change it to `minor`). 

Apps that relied heavily on being installed `perMachine` won't work unless they specify `perMachine`. I don't have an example at the top of my head but I thought I'd be cautious and treat it as a breaking.

Reply:
doesn't it make more sense to keep it enabled by default then?

Reply:
I prefer perUser as default but if it can break any kind of app, we should keep the per machine as default.

Reply:
it has a slight chance to break some apps but I think `perUser` should be the default, even if it means this PR shouldn't be merged in v1

Reply:
while i can't think of anything really where that would break something, but what about merging this pr with perMachine as default and changing the default with v2?

Reply:
I have to move my glow stick for you guys. 
I think this is a very important function and it will be very helpul if it can be merged and published as soon as possible.